src/utils.jl

```diff
@@ -102,8 +102,8 @@ it returns
 ```math
 \\sum_{𝐤} |û_{𝐤}|² L_x L_y = \\int u(𝐱)² \\, 𝖽x 𝖽y \\,,
 ```
-where ``û_{𝐤} =`` `uh` ``/(`` `grid.nx` ``e^{- i 𝐤 ⋅ 𝐱₀})``, with ``𝐱₀`` the vector with components
-the left-most position in each direction.
+where ``û_{𝐤} =`` `uh` `` / (`` `grid.nx` ``e^{i 𝐤 ⋅ 𝐱₀})``, with ``𝐱₀`` the vector with components
+the left-most position in each direction, e.g., ``𝐱₀ = (`` `grid.x[1]`, `grid.y[1]` ``)``.
```

Comment:
Also maybe ``û_{𝐤} =`` `uh` `` / (`` `grid.nx` ``e^{i 𝐤 ⋅ 𝐱₀})`` could be formatted so the parentheses are both inside the code block or even just as one code block,  ``û_{𝐤} = uh  / ( grid.nx e^{i 𝐤 ⋅ 𝐱₀})``?